earthmover/nodes/destination.py

```diff
@@ -104,7 +104,9 @@ def execute(self, **kwargs):
             if self.header:
                 fp.write(self.header)
 
-            self.data.apply(lambda row: fp.write(row + '\n'), meta=pd.Series('string')).compute()
+            for partition in self.data.partitions:
+                fp.writelines(partition.compute())
+                partition = None  # Remove partition from memory immediately after write.
```

Comment:
Why is this necessary?

Reply:
That's right. If we don't do this, `partition` will linger in memory at least until the end of the execute. I don't want any residual objects to be saved.